container/deploy_openqa_containers.sh

```diff
@@ -0,0 +1,228 @@
+  wget https://raw.githubusercontent.com/os-autoinst/openQA/master/container/webui/conf/database.ini  >/dev/null
+  wget https://raw.githubusercontent.com/os-autoinst/openQA/master/container/openqa_data/data.template/conf/client.conf >/dev/null
+  popd || exit 1
+
+  openssl req -newkey rsa:4096 -x509 -sha256 -days 365 -nodes -subj '/CN=www.mydom.com/O=My Company Name LTD./C=DE' \
+    -out data/certs/ssl.crt/server.crt -keyout data/certs/ssl.key/server.key >/dev/null
+
+  cp data/certs/ssl.crt/server.crt data/certs/ssl.crt/ca.crt
+
+  volumes="-v $(pwd)/data:/data"
+  certificates="-v $(pwd)/data/certs/ssl.crt:/etc/apache2/ssl.crt -v $(pwd)/data/certs/ssl.key:/etc/apache2/ssl.key"
+
+  [[ $FORCE -eq 1 ]] && set +e && docker rm -f "$CONT_NAME" 2>/dev/null 2>&1; set -e
+  docker run -d --network "$NETWORK" $volumes $certificates -p $PORT:80 --net-alias=openqa_webui --name "$CONT_NAME" "$WEBUI_IMAGE" || exit 1
+  
+  wait_for_container_ready "$CONT_NAME"
+  wait_for_container_appropiate_logs "$CONT_NAME" 'Listening at "http://127.0.0.1:9527"'
+  wait_for_container_appropiate_logs "$CONT_NAME" 'Listening at "http://127.0.0.1:9528"'
+  wait_for_container_appropiate_logs "$CONT_NAME" 'Listening at "http://127.0.0.1:9529"'
+  wait_for_container_appropiate_logs "$CONT_NAME" 'Web application available at'
+  
+  echo "Next step (worker), stand-up of worker container"
+  echo "You have to provide an API key/secret. Access to the UI to create a pair at http://localhost:$PORT"
+}
+
+function worker {
+  [[ -z $KEY ]] || [[ -z $SECRET ]] && echo "--key and --secret are required" && exit 1
+
+  [[ ! -d data ]] && [[ ! -d data/conf ]] && echo "Please execute first prepare command" && exit 1
+
+  CONT_NAME="${PREFIX}_worker"
+
+  pushd .
+  cd data/conf
+  wget https://raw.githubusercontent.com/os-autoinst/openQA/master/container/openqa_data/data.template/conf/workers.ini >/dev/null 2>/dev/null
+  popd || exit 1
+
+  echo -e "[openqa_webui]\nkey = $KEY\nsecret = $SECRET" > data/conf/client.conf
+
+  [[ $FORCE -eq 1 ]] && set +e && docker rm -f "$CONT_NAME" >/dev/null 2>&1 && set -e
+  docker run -d --network "$NETWORK" -v "$(pwd)/data:/data" --device=/dev/kvm --privileged --name "$CONT_NAME" "$WORKER_IMAGE" >/dev/null
+
+  wait_for_container_ready "$CONT_NAME"
+  wait_for_container_appropiate_logs "$CONT_NAME" 'Registering with openQA'  
+  echo -n "Checking if API credentials are correct.. ."
+  wait_for_container_appropiate_logs "$CONT_NAME" 'Registered and connected via websockets with openQA' 4
+
+  echo "All ready. Enjoy!"
+  echo "You have to provide your own tests or the upstream tests and neeles and store them in the directory data/tests"
+  echo "e.g. of usage: docker exec -ti ${PREFIX}_webui /usr/share/openqa/script/clone_job.pl --apikey $KEY --apisecret $SECRET <upstream_job>"
+}
+
+function clean {
+  set +e
+  docker rm -f "${PREFIX}_worker"
+  docker rm -f "${PREFIX}_webui"
+  docker rm -f "${PREFIX}_db"
```

Comment:
I suppose this should be possible:
```suggestion
  docker rm -f "${PREFIX}_worker" "${PREFIX}_webui" "${PREFIX}_db"
```
Otherwise you could use a loop.